verl/utils/torch_functional.py

```diff
@@ -289,6 +290,17 @@ def postprocess_data(
     return input_ids, attention_mask
 
 
+def tokenize_and_postprocess_data(prompt: str, tokenizer: PreTrainedTokenizer, max_length: int, pad_token_id: int, left_pad=True, truncation="error"):
+    """
+    input_data is the output from tokenizer.
```

Comment:
please briefly describe what post processing is done, and same for the postprocess_data api